### ISSUE TITLE
Component/integration testing for Grant Access Page

### ISSUE THREAD

**Issue by MCatherine1994** [bcgov/nr-forests-access-management#717]
**Describe the task**
Create integration test for the grant access page. We already have separate ticket to test user domain selection #1082, and username input #1083, but we still need to test the validation error for username input as the validation is set at form level. 

**Acceptance Criteria**
- [ ] test the functionality for all elements in the grant access page
     - [ ] test the breadcrumb, page title, sub title, form fields are rendered on the page, submit button is disabled
     - [ ] test the validation error and message for user id field
         - [ ]  test the invalid input, user id is invalid (length<2), the error message is shown, the button is disabled
         - [ ]  test this field is required
     - [ ] test the dropdown selection for role
     - [ ] test select an abstract role will show the forest client number field
     - [ ] test the validation error and message for forest client number field
         - [ ]  test the invalid input, forest client number is invalid (length<8), the error message is shown, the button is disabled
     - [ ] test submit button will be enabled if got all required fields
     - [ ] test click on cancel button, page will be redirected
- [ ] increase the code coverage for the grant access page

Note: 
- test for forest client number field is in another ticket #970

**Comment by OlgaLiber2** [comment]
@MCatherine1994 is this ticket still relevant?

**Comment by MCatherine1994** [comment]
yes, this is the testing ticket for the grant access page. But it's low priority since we will do more changes on that page.

**Comment by OlgaLiber2** [comment]
@MCatherine1994  is this ticket still relevant?
